ui/src/models/workflows.ts

```diff
@@ -766,6 +766,11 @@ export interface WorkflowStatus {
      * Conditions is a list of WorkflowConditions
      */
     conditions?: WorkflowCondition[];
+
+    /**
+     * StoredWorkflowTemplateSpec is a Workflow Spec of top level WorkflowTemplate.
```

Comment:
The same comment as the `*_types.go` file
```suggestion
     * StoredWorkflowSpec stores the top level WorkflowTemplate spec
```